pkgs/by-name/hd/hdrop/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
+  scdoc,
+  coreutils,
+  util-linux,
+  jq,
+  libnotify,
+  hyprland ? null,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "hdrop";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "Schweber";
+    repo = "hdrop";
+    rev = "9c29164";
```

Use full hash, or use something like 
```suggestion
    rev = v${version};
```